### ISSUE TITLE
为什么nvs不火？

### ISSUE THREAD

**Issue by wsz7777** [jasongin/nvs#130]
这么优秀的一个node版本管理器，为何无人问津，才有692颗星？？？？？

**Comment by JimPatterson** [comment]
I agree, more people should use nvs.

**Comment by jasongin** [comment]
_Rough translation: It's a good version manager, why isn't it more popular?_

I'm glad you like it. Feel free to promote in any ways you feel appropriate: blog, tweet, tell you colleagues, etc.

But I'm going to close this issue since it's not a bug.

**Comment by wsz7777** [comment]
my friends use it now.
hope it better .
do you want use typescript code it ?

**Comment by jasongin** [comment]
> do you want use typescript code it ?

I use TypeScript a lot in my job, and like it very much. When I started this project I chose not to use TypeScript for a few reasons, and I still think that was the right choice.
  - It is convenient to be able to clone the repo and run the tool right away, without any required build step or external dependencies.
  - This is not a library that publishes APIs for others to consume, that would benefit from type definitions.
  - This is a relatively small project, without a lot of complex code or data structures that would benefit much from TS capabilities.

Note I _have_ set up this project to use TypeScript to perform type-checking on the JS code:
```
npm install
npx tsc
```

(Actually running that just now I see a recent contribution introduced a few issues. I should fix those.)

To get more type-checking it would be possible to add more jsdoc-style type annotations, but I haven't put in the effort.
